### PR TITLE
chattr: add page

### PR DESCRIPTION
**Pull request by rozuur**
Added common use case for [chattr](http://linux.die.net/man/1/chattr).

### CODE REVIEW

File: pages/linux/chattr.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# chattr
+
+> Change attributes of files or folders.
+
+- Make a file or folder immutable to changes and deletion, even by superuser:
+
+`chattr +i {{path}}`
+
+- Undo previous changes to make a file or folder mutable:
```
Comment (ostera) [inline pages/linux/chattr.md]: Shouldn't this be just `make a file or folder mutable`? It doesn't really have anything to do with the previous change (file could have been created as immutable).

Comment (rozuur) [inline pages/linux/chattr.md]: will make the change and resubmit.

**Comment by ostera** [comment]
:+1:

**Comment by rozuur** [comment]
Thanks a lot

**Comment by ostera** [comment]
Thank you for your contribution @rozuur :)

**Pull request merged**